WordPress/Classes/ViewRelated/Reader/ReaderDetailViewController.swift

```diff
@@ -32,6 +32,7 @@ open class ReaderDetailViewController: UIViewController, UIViewControllerRestora
     @IBOutlet fileprivate weak var commentButton: UIButton!
     @IBOutlet fileprivate weak var likeButton: UIButton!
     @IBOutlet fileprivate weak var footerViewHeightConstraint: NSLayoutConstraint!
+    fileprivate var relatedPostsController: RelatedPostsViewController?
```

Comment:
I don't think specifying `weak` is the right thing to do here.  There would be nothing else retaining the controller.